docker/nvidia/setup_nvidia_docker_compose.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import requests
+import yaml
+
+# query nvidia docker plugin for the command-line parameters to use with the
+# `docker run` command
+response = requests.get('http://localhost:3476/docker/cli/json')
+docker_cli_params = response.json()
```

Comment:
Add graceful handling of the failure. For instance, if you don't have NVidia docker plugin installed, here is what you get right now:
```
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=3476): Max retries exceeded with url: /docker/cli/json (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f921483e490>: Failed to establish a new connection: [Errno 111] Connection refused',))
```
What you want is to provide a user/developer with meaningful message.